examples/component/packages/my-component/README.md

```diff
@@ -0,0 +1,3 @@
+# Example `@example/my-component`
+
+...
```

Comment:
Could use a little help here.

Reply:
I would expect this to be like an open-source project, right? So some usage examples, props API, etc.